File: official/docs/php/current/addresses/verify-failure.php
<?php

$client = new \EasyPost\EasyPostClient(getenv('EASYPOST_API_KEY'));

$address = $client->address->create([
    'street1' => '000 unknown street',
    'city'    => 'Not a City',
    'state'   => 'ZZ',
    'zip'     => '00001',
    'country' => 'US',
    'email' => 'test@example.com',
    'phone'   => '5555555555',
]);

echo $address;

File: official/docs/golang/v3/addresses/verify-failure.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v3"
)

func main() {
	client := easypost.New("EASYPOST_API_KEY")

	address, _ := client.CreateAddress(
		&easypost.Address{
			Street1: "UNDELIVERABLE ST",
			City:    "SAN FRANCISCO",
			State:   "CA",
			Zip:     "94104",
			Country: "US",
			Company: "EasyPost",
			Phone:   "415-123-4567",
		},
		&easypost.CreateAddressOptions{},
	)

	fmt.Println(address)
}
